subsys/net/lib/lwm2m/lwm2m_registry.c

```diff
@@ -52,11 +53,21 @@ sys_slist_t *lwm2m_engine_obj_inst_list(void) { return &engine_obj_inst_list; }
 
 void lwm2m_register_obj(struct lwm2m_engine_obj *obj)
 {
+#if defined(CONFIG_LWM2M_ACCESS_CONTROL_ENABLE)
+	/* If bootstrap, then bootstrap server should create the ac obj instances */
+#if !IS_ENABLED(CONFIG_LWM2M_RD_CLIENT_SUPPORT_BOOTSTRAP)
+	/* -1, to use the default */
+	access_control_add_obj(obj->obj_id, -1);
```

Comment:
`-1` is not very descriptive when looking over the code w/o reaching to the docmunetation, perhaps you could define some symbol for the magic number?